### PR TITLE
Mn 2017 10 creator automatically co worker

### PR DESCRIPTION
**Pull request by MagdaN**
When an idea is created the creator is automatically added to co_workers, but can be removed afterwards (but would still be displayed in the team list). That's what we had in mind, right?

### CODE REVIEW

**Comment by gereonrahnfeld** [comment]
Which one is the co-workers list and which one the team list? Is this not the same?

**Review by fuzzylogic2000 — COMMENTED** [review]
Nice! But I think, we should go the whole way and not use the creator for permissions or anywhere anymore. What do you think, @slomo

**Comment by MagdaN** [comment]
Yes, but we should make ensure - if we remove the creator from the model (or at least the permissions) - that not all users can be removed from the co_workers list, otherwise there would be no user left that is allowed to e.g. edit the idea and add co_workers.

**Comment by slomo** [comment]
@gereonrahnfeld This kind of confusion happens if a feature is named differently in the UI than in the code. :smile:. It is now called `co_workers` in the code, but `team` in ui. We intend to fix this in the long run, but for now we stick with coworkers.

**Comment by slomo** [comment]
I would probably also update the `co_workers? method in the ideasketch factory, so that the creator is automatically added to the coworkers.

File: apps/ideas/views.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def done(self, form_list, **kwargs):
                 if field not in special_fields
                 }
         )
-
+        idea_sketch.co_workers.add(self.request.user)
```
Comment (slomo) [inline apps/ideas/views.py]: Maybe we should add an transaction here, so that we don't create an idea_sketch without coworker accidentally.

**Comment by slomo** [comment]
As discussed with @fuzzylogic2000 we might also consider to extend the wizard with a check that at least one co worker is always left in the proposal. In addition also a check in the models clean method could be useful, so that the admin can't do it accidentally.

**Comment by MagdaN** [comment]
@slomo I added the atomic decorator two both done methods (IdeaSketchWizard and ProposalWizard). That should do the job, right?

**Pull request closed without merging**